woocommerce/woocommerce-gutenberg-products-block
# Remove unused Verticals client PHP tests

## What

Removes unused Verticals client PHP tests.

## Why

The class was removed in https://github.com/woocommerce/woocommerce-blocks/pull/11280/ but the test was not.

The class was loaded from WC Core instead. Since updating to WC 8.3 the tests are failing because the class no longer exists in our plugin or in core.



## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Ensure unit tests pass in CI.


## Screenshots or screencast



| Before | After |
| ------ | ----- |
|        |       |

## WooCommerce Visibility


Required:


## Checklist

Required:


Conditional:


## Changelog


> Skipping
